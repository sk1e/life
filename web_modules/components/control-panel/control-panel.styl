.control-panel
  margin auto
  display flex
  justify-content center
  align-items center
  
  & .input:not(:first-child)
    margin-left 1rem
    
