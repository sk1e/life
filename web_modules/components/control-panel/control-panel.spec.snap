exports[`ControlPanel renders correctly`] = `
<div
  className="control-panel">
  <Connect(sizeInputFieldContainer) />
  <Connect(sizeInputFieldContainer) />
  <Connect(PlayButton) />
</div>
`;
